yt_dlp/extractor/twitter.py

```diff
@@ -90,6 +90,9 @@ def _call_api(self, path, video_id, query={}):
         headers = {
             'Authorization': 'Bearer AAAAAAAAAAAAAAAAAAAAAPYXBAAAAAAACLXUNDekMxqa8h%2F40K4moUkGsoc%3DTYfbDKbT3jJPCEVnMYqilB28NHfOPqkca3qaAxGfsyKCs0wRbw',
         }
+        token = self._get_cookies(self._API_BASE + path).get('ct0')
```

Comment:
cookies are domain-specific, not url specific. So this is enough
```suggestion
        token = self._get_cookies(self._API_BASE).get('ct0')
```